packages/core-components/src/layout/Sidebar/Bar.tsx

```diff
@@ -17,10 +17,12 @@
 import { makeStyles } from '@material-ui/core/styles';
 import useMediaQuery from '@material-ui/core/useMediaQuery';
 import clsx from 'clsx';
-import React, { useRef, useState, useContext, PropsWithChildren } from 'react';
+import React, { useState, useContext, PropsWithChildren, useRef } from 'react';
 import { sidebarConfig, SidebarContext } from './config';
 import { BackstageTheme } from '@backstage/theme';
 import { SidebarPinStateContext } from './Page';
+import DoubleArrowRight from './icons/DoubleArrowRight';
+import DoubleArrowLeft from './icons/DoubleArrowLeft';
```

Comment:
Some of them I wasn't able to find and some I could not import even though they should be in material-ui. Getting some confirmation on this, will get back to you about it soon

Reply:
Alright, thanks! We'd like to avoid adding any additional items, both because MUI already offers a rich set of icons, but also because we'd need to make sure licensing of any added items is done correctly, which can be a can of worms.

Reply:
> 

![image](https://user-images.githubusercontent.com/72443252/143582655-e7c7fa17-4882-4bd9-a554-3491b7203dd0.png)
When I try to import many of the icons that should be in @material-ui/icons, I get this error. Can't seem to figure out why because the documentation for the material-ui package say it should be there.

Reply:
Those are from the docs for v5 of material-ui - we are sadly still on v4. The imports will be on the form `@material-ui/icons/X`. The docs are here:

https://v4.mui.com/

And the icons are here

https://v4.mui.com/components/material-icons/

Reply:
Got it, thank you!! Not an issue, I replaced most of the icons with ones in v4 as they're only really used in the storybook. I created a double arrow from scratch with the existing single arrows from mui so it still matches the design 🌟

Reply:
@Rugvip I believe the latest commit should resolve this issue now